tests/platform_tests/conftest.py

```diff
@@ -18,6 +18,11 @@
 TEMPLATES_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), "templates")
 FMT = "%b %d %H:%M:%S.%f"
 FMT_SHORT = "%b %d %H:%M:%S"
+SMALL_DISK_SKUS = [
+    "Arista-7060CX-32S-C32",
+    "Arista-7060CX-32S-Q32"
+]
```

Comment:
Can you try "df -h | grep "/var/log", if it is tmpfs, then apply the backup?

Reply:
I liked that approach too. But the problem is that for upgrade scenario (for eg. 201811), `/var/log` can be mounted on `/dev/***`, and when reboots to a large image (for eg. 202012), gets mounted on `tmpfs`.

In that case I _think_ we still need SKUs listed separately.